packages/backend/src/util/IsomorphicRequest.ts

```diff
@@ -0,0 +1,27 @@
+import { parse } from 'cookie';
+
+import runtime from '../runtime';
+
+type IsomorphicRequestOptions = (Request: Request, Headers: Headers) => Request;
+
+export const createIsomorphicRequest = (cb: IsomorphicRequestOptions): Request => {
```

Comment:
❓ Why are we calling this an IsomorphicRequest since it's just the global.Request?

Reply:
I recommended the IsomorphicRequest name because it's a Request that can run on any platform, even platforms that natively do not support `global.Request` (older node runtimes etc).

Would `createRequest` make more sense to you? I didn't choose this one because it doesn't make the difference between `new Request` and `createRequest` apparent. Happy to change the name though :)